### PR TITLE
chore(bump): v3.20.0-alpha.0

### PR DESCRIPTION
**Pull request by sfoster1**
Bump PR for 3.20.0 alpha.0

### CODE REVIEW

**Review by Kadee80 — APPROVED** [review]
🧊

File: api/release-notes.md [hunk 1]
```diff
@@ -1,32 +1,23 @@
...
-  protocols
-- Load a more realistic deck transform when simulating protocols via
-  `opentrons_simulate` to avoid `LabwareHeightErrors` during command line
-  simulation ([#5908](https://github.com/opentrons/opentrons/issues/5908))
-  ([03757d9](https://github.com/opentrons/opentrons/commit/03757d9))
-- Fix an issue causing plate temperature uniformity errors if the Thermocycler
-  lid is opened immediately after a `set_block_temperature` ([#5602](https://github.com/opentrons/opentrons/issues/5602))  
-- Fix an issue causing incorrect display of delay times in Protocol Designer
-  protocols ([#5414](https://github.com/opentrons/opentrons/issues/5414)) 
+- Instrument offsets (the calibration values saved from Pipette Calibration before protocol runs) are now cleared before running Deck Calibration, through either the in-app or CLI tools. This fixes an issue where incorrect instrument offsets would cause deck calibrations to be incorrect.
```
Comment (Laura-Danielle) [inline api/release-notes.md:13]: Should we also mention how you no longer need to restart the robot for deck calibration?

Comment (sfoster1) [reply api/release-notes.md:13]: @Laura-Danielle that's in the app release notes since it's an app change

**Review by mcous — APPROVED** [review]

**Review by bndo — APPROVED** [review]
LGTM 🕺

**Pull request merged**